common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -9,7 +9,7 @@
 <div class="choicegroup capa_inputtype" id="inputtype_${id}">
     <fieldset ${describedby_html}>
       % if response_data['label']:
-        <legend id="${id}-legend" class="response-fieldset-legend field-group-hd">${response_data['label']}</legend>
+        <legend id="legend-${id}" class="response-fieldset-legend field-group-hd">${response_data['label']}</legend>
```

Comment:
@arizzitano Take a look at this line in capa_problem.py (538):

problem_legend_id = str(choicegroup.get('id')) + '-legend'